EMaligner/utils.py

```diff
@@ -185,33 +186,40 @@ def get_tileids_and_tforms(stack, tform_name, zvals, fullsize=False, order=2):
 
 def get_matches(iId, jId, collection, dbconnection):
     if collection['db_interface'] == 'render':
+        mlist = []
         if iId == jId:
-            matches = renderapi.pointmatch.get_matches_within_group(
-                    collection['name'],
-                    iId,
-                    owner=collection['owner'],
-                    render=dbconnection)
+            for name in collection['name']:
+                mlist.append(renderapi.pointmatch.get_matches_within_group(
+                        name,
+                        iId,
+                        owner=collection['owner'],
+                        render=dbconnection))
         else:
-            matches = renderapi.pointmatch.get_matches_from_group_to_group(
-                    collection['name'],
-                    iId,
-                    jId,
-                    owner=collection['owner'],
-                    render=dbconnection)
-        matches = np.array(matches)
+            for name in collection['name']:
+                mlist.append(
+                        renderapi.pointmatch.get_matches_from_group_to_group(
+                            name,
+                            iId,
+                            jId,
+                            owner=collection['owner'],
+                            render=dbconnection))
+        matches = np.concatenate(mlist)
```

Comment:
It looks like these used to return lists but now return numpy arrays.  Any reason not to use list operations like extend here?

Reply:
It looks like no, no reason. I suppose lists would be more in line with renderapi get_matches...()